core/src/main/java/com/datastax/oss/driver/internal/core/config/typesafe/DefaultDriverConfigLoader.java

```diff
@@ -55,7 +55,8 @@
   public static final Supplier<Config> DEFAULT_CONFIG_SUPPLIER =
       () -> {
         ConfigFactory.invalidateCaches();
-        return ConfigFactory.load().getConfig(DEFAULT_ROOT_PATH);
+        return ConfigFactory.load(DriverConfigLoader.DRIVER_CLASS_LOADER)
```

Comment:
Isn't this going to use `DRIVER_CLASS_LOADER` to load both application and reference files? Maybe we need to make this identical to `Default ProgrammaticDriverConfigLoaderBuilder.DEFAULT_FALLBACK_SUPPLIER`.